README.md

```diff
@@ -27,67 +27,140 @@ collection <- "<collection_name>"
 mongodb_uri <- "<mongodb://uri>"
 
 # Création d'une tâche d'apprentissage
+# C'est un objet de type "S3", donc une liste dans laquelle sont stockées les 
+# informations de la tâche d'apprentissage
 task <- sf_task(
   mongodb_uri,
   database,
   collection
 )
 
-last_batch <- "2005_2" # Nom du dernier batch importé
-sample_size <- 10000 # Échantillonnage pour l'entraînement de 10.000 couples (siret x période). 
-# (en réalité 800.000 en production)
+last_batch <- "2009_5" # Nom du dernier batch importé
 
-# Chargement des données historiques
+# Échantillonnage aléatoire pour l'entraînement de 10.000 
+# couples (siret x période).  (en réalité 800.000 en production)
+sample_size <- 10000 
+
+# Chargement des données historiques (par défaut, 2015 et 2016)
 task <- load_hist_data(
   task = task,
   batch = last_batch,
   subsample = sample_size
 )
 
-# On entraîne sur toute la base.
-task <- split_data(task, resampling_strategy = "none") 
 
 # Utilitaire pour lister les champs
 fields <- get_fields(training = TRUE) 
 
-# Pipeline de préparation des données. task <- prepare(
+# Si l'on entraîne sur toute la base, pas besoin de créer d'échantillons de 
+# test. Se référer au script de test pour voir comment faire. 
+
+# prepare spécifie une suite d'opérations sur les données, sous la forme d'un
+# `mlr3pipelines::Graph` ou assimilable. 
+# Les opérations ne seront effectuées qu'au moment de l'entraînement.
+task <- prepare(
   task,
-  training_fields = training_fields
+  training_fields = fields
 )
 
+# Si l'on veut tout de même consulter l'effet de la préparation sans entraîner 
+# de modèle, on peut utiliser:
+prepared_data <- get_prepared_data(task)
```

Comment:
> prepared_data <- get_prepared_data(task)
Error in train_or_test %in% c("train", "test") : 
  argument "train_or_test" is missing, with no default

Reply:
Et si je fais:

`prepared_data <- get_prepared_data(task, train_or_test = "train")`
Error in task[["mlr3rsmp"]]$train_set(1) : attempt to apply non-function

Reply:
Mmh, mea culpa, les modifications étaient restés coincés dans une PR en draft... PR #51 
Je l'ai mise prête à être mergée, elle devrait résoudre ce pb.

Reply:
PS: Il va sans doute y avoir des nouvelles modifications imminentes sur cette fonction